crates/bevy_ui/src/lib.rs

```diff
@@ -229,3 +182,51 @@ impl Plugin for UiPlugin {
         render_app.init_resource::<UiPipeline>();
     }
 }
+
+/// A function that should be called from [`UiPlugin::build`] when [`bevy_text`] is enabled.
+#[cfg(feature = "bevy_text")]
+fn build_text_interop(app: &mut App) {
+    use bevy_text::TextLayoutInfo;
+    use crate::widget::TextFlags;
+
+    app.register_type::<TextLayoutInfo>()
+        .register_type::<TextFlags>();
+
+    // add these systems to front because these must run before transform update systems
```

Comment:
Does anyone know why this comment is here? Is it still being upheld since I moved all the logic to a function?

Reply:
We should remove it now.